src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerConfigTest.java

```diff
@@ -95,27 +102,42 @@ static void setupKernel() throws InterruptedException,
         assertTrue(logManagerRunning.await(10, TimeUnit.SECONDS));
     }
 
-    @AfterAll
-    static void afterAll() {
+    @BeforeEach
+    void beforeEach(ExtensionContext context)
+            throws DeviceConfigurationException, URISyntaxException, IOException, InterruptedException {
+        ignoreExceptionOfType(context, TLSAuthException.class);
```

Comment:
Is this really needed for all the tests - We should just add it to the tests when they throw that type of exception and we expect it. Otherwise this just hides some real errors

Reply:
This test was designed not to process any files. But now the active file is processed.